### PR TITLE
Add missing tests for issue #71

### PR DESCRIPTION
**Pull request by tomalec**
Problem: neither href nor model was set for the inner imported-template if the partial/viewModel property was attached before CE upgrade.

### CODE REVIEW

**Comment by tomalec** [comment]
Please note, that failing tests are exactly the once that were failing for 4.0.1.

So it adds 6 new tests that were failing in 4.0.0, and are fine in 4.0.1

File: test/partialAttribute/nested-html/property-set-before-CEupgrade.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                },
+                'scope3': {
+                    'doesItWork': 'works!'
+                }
+            };
+        };
+        const MERGED_URL = '/sc/htmlmerger?scope1=scope1Html&scope2=scope2Html%3B%2C%2F%3F%3A%40%26%3D%2B%24&scope3=';
+
+        describe('once element with viewModel property set before upgrade is upgraded', function () {
+            var container = document.querySelector('#container'),
```
Comment (alshakero) [inline test/partialAttribute/nested-html/property-set-before-CEupgrade.html]: It seems `container` is never used.

File: test/partialAttribute/nested-html/property-set-before-CEupgrade.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                disconnectedSCInclude, disconnectedPartial, connectedSCInclude, connectedPartial, importedTemplate;
+            before(function(done) {
+                disconnectedSCInclude = document.createElement('starcounter-include');
+                disconnectedSCInclude.viewModel = disconnectedPartial = createPartial();
+                connectedSCInclude = document.querySelector('#connected-fixture > starcounter-include');
+                connectedSCInclude.viewModel = connectedPartial = createPartial();
+                const link = document.createElement('link');
```
Comment (alshakero) [inline test/partialAttribute/nested-html/property-set-before-CEupgrade.html:53]: such a smart technique to test!

**Review by alshakero — APPROVED** [review]
One simple issue. You can merge after fixing it.

File: test/partialAttribute/nested-html/property-set-before-CEupgrade.html (hunk 2 above)
Comment (tomalec) [reply test/partialAttribute/nested-html/property-set-before-CEupgrade.html:53]: which one?

Comment (alshakero) [reply test/partialAttribute/nested-html/property-set-before-CEupgrade.html:53]: I mean importing `starcounter-include` later this way to mimic async behaviour.

Comment (tomalec) [reply test/partialAttribute/nested-html/property-set-before-CEupgrade.html:53]: Well, it's actually "the behavior" ;) 
1. CE Element  is attached (as HTMlUnknownElement)
2. you can fiddle it, and use it as any other HTMLElement,
3. CE definition is loaded,
4. Elements are upgraded

**Comment by warpech** [comment]
Setting as “Finished” because it is:
- released (with release notes) https://github.com/Starcounter/starcounter-include/releases/tag/4.1.0
- included in a Finished release of SCF https://github.com/Starcounter/StarcounterClientFiles/releases/tag/2.1.0

**Pull request merged**